### PR TITLE
tag queries are for full words, not substrings

### PR DESCRIPTION
**Pull request by cindytung**
✌️

### CODE REVIEW

File: cal/cal/models.py [hunk 1]
```diff
@@ -655,7 +655,7 @@ def query(self, calendar_ids=None, start=None, end=None):
 
         querysets = [
                 GEvent.objects
-                .filter(calendar=calendar, name__icontains=keyword)
+                .filter(calendar=calendar, name__regex=r'\b'+keyword+r'\b')
```
Comment (MaxFangX) [inline cal/cal/models.py]: omfg I never knew you could do this. You're amazing

**Review by MaxFangX — APPROVED** [review]
Can you confirm that this is still case-insensitive? Otherwise feel free to merge (:

File: cal/cal/models.py (hunk 1 above)
Comment (cindytung) [reply cal/cal/models.py]: ahh you're right, I didn't make it case insensitive. still new to regex :)

File: cal/cal/models.py [hunk 2]
```diff
@@ -655,7 +655,7 @@ def query(self, calendar_ids=None, start=None, end=None):
 
         querysets = [
                 GEvent.objects
-                .filter(calendar=calendar, name__regex=r'\b'+keyword+r'\b')
+                .filter(calendar=calendar, name__regex=r'\b(?i)'+keyword+r'\b')
```
Comment (jamesuejio) [inline cal/cal/models.py]: Impressive. I think anything that deals with regex is magic

**Review by jamesuejio — APPROVED** [review]

**Pull request merged**